tests/derive/test_assign_outer_postcode.py

```diff
@@ -5,7 +5,7 @@
 
 def test_assign_outer_postcode(spark_session):
     expected_df = spark_session.createDataFrame(
-        data=[("E26 4LB", "E26"), ("E6 2PC", "E6"), ("IG120DB", "IG12")],
+        data=[("E26 4LB", "E26"), ("IG14  2PC", "IG14"), ("IG120DB", "IG12")],
```

Comment:
Would only see two spaces when the outward part is 2-character
```suggestion
        data=[("E26 4LB", "E26"), ("E6  2PC", "E6"), ("IG120DB", "IG12")],
```